server/container/guice/protocols/jmap/src/main/java/org/apache/james/jmap/rfc8621/RFC8621MethodsModule.java

```diff
@@ -137,6 +138,7 @@ protected void configure() {
         bind(WebPushClient.class).to(DefaultWebPushClient.class);
 
         bind(UploadService.class).to(UploadServiceDefaultImpl.class).in(Scopes.SINGLETON);
+        bind(JMAPCurrentUploadUsageRecomputator.class).in(Scopes.SINGLETON);
```

Comment:
Recomputator do not exist in english.

Please let's use words that exist.

JMAPCurrentUploadUsageFactory ? JMAPCurrentUploadUsageComputer ? Something else?